pkg/operator/ceph/cluster/cluster.go

```diff
@@ -194,6 +181,18 @@ func (c *ClusterController) initializeCluster(cluster *cluster) error {
 			return errors.Wrap(err, "failed to configure external ceph cluster")
 		}
 	} else {
+		clusterInfo, _, _, err := mon.LoadClusterInfo(c.context, c.OpManagerCtx, cluster.Namespace)
```

Comment:
Does moving this just avoid an unnecessary log message for external clusters? Or any other side effect?